openmrs/apps/ot/app.json

```diff
@@ -0,0 +1,20 @@
+{
+    "id": "bahmni.ot.scheduling",
+    "extensionPoints": [
```

Comment:
Remove extension points if not needed.

Reply:
ok Soorya, it is not needed as of now will remove it.